tests/interpreter/cases/builtins/bitwise/tests.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+cases:
+  - note: bits-error
+    data: {}
+    modules:
+      - |
+        package test
+
+        x = bits.and(1, "str")
```

Comment:
Also lock down with a test for `and(string, number)` as well as with more than two parameters.